backend/copr_backend/rpm_builds.py

```diff
@@ -150,6 +151,25 @@ def __init__(self, architecture, limit):
         )
 
 
+class UserSSHLimit(HashWorkerLimit):
+    """
+    Limit the number of builders that allow user SSH
+    """
+    def __init__(self, _hasher, limit):
+        def predicate(x):
+            # We don't allow user SSH for SRPM builds, return unique string
+            # making this unlimited
+            if x.source_build:
+                return str(uuid.uuid4())
```

Comment:
I think we can simply return None, and the HashWorkerLimit will not count with those.
Btw., I noticed this because of the memory leak we have in StringCounter() (at least with random strings like this).